{
	"title": "CVE-2019-19911",
	"cve": "CVE-2019-19911",
	"notes": [
		{
			"type": "Description",
			"text": "There is a DoS vulnerability in Pillow before 6.2.2 caused by FpxImagePlugin.py calling the range function on an unvalidated 32-bit integer if the number of bands is large. On Windows running 32-bit Python, this results in an OverflowError or MemoryError due to the 2 GB limit. However, on Linux running 64-bit Python this results in the process being terminated by the OOM killer."
		},
		{
			"title": "Published",
			"type": "Other",
			"text": "2020-01-05"
		},
		{
			"title": "Modified",
			"type": "Other",
			"text": "2020-02-22"
		}
	],
	"references": [
		{
			"url": "https://pillow.readthedocs.io/en/stable/releasenotes/6.2.2.html",
			"description": "CONFIRM:https://pillow.readthedocs.io/en/stable/releasenotes/6.2.2.html"
		},
		{
			"url": "https://www.debian.org/security/2020/dsa-4631",
			"description": "DEBIAN:DSA-4631"
		},
		{
			"url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/3DUMIBUYGJRAVJCTFUWBRLVQKOUTVX5P/",
			"description": "FEDORA:FEDORA-2020-5cdbb19cca"
		},
		{
			"url": "https://usn.ubuntu.com/4272-1/",
			"description": "UBUNTU:USN-4272-1"
		}
	]
}
